modules/appeals_api/app/swagger/appealable_issues/v0/swagger.json

```diff
@@ -200,9 +200,10 @@
                 "example": {
                   "errors": [
                     {
```

Comment:
In these docs you can see the results of relying on `Common::Exceptions` to render our errors - the default body contents of those `Common::Exceptions` are slightly different from the ones we've returned so far